deploy/backplane/nvidia-gpu/20-nvidia-gpu-readers.ClusterRoleBinding.yml

```diff
@@ -0,0 +1,11 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: backplane-nvidia-gpu-readers
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: backplane-nvidia-gpu-readers-cluster
```

Comment:
Do you need view access to the whole cluster, including customer namespaces?

Reply:
This role will only be used by the addon's engineers to support pilot customers. Outside of that context I would say no, but the expanded view access would be needed for hands-on customer support. Again that only applies since this role is not the same that would be used by SRE's for LA/GA support.

Reply:
@cblecker did that explanation answer your question or are there still concerns about the scope of view access?

Reply:
This sounds like better suited for privilege escalation which has a process behind it to ensure such access is justified.  Since there is not going to be a touch point to triage this access when transitioning out of pilot I would like to hear how escalation is insufficient.  

https://github.com/openshift/ops-sop/blob/master/v4/howto/backplane-elevate-privileges.md